<launch>
    <param name="use_sim_time" value="true"/>

    <node pkg="tf" type="static_transform_publisher" name="base_link_to_camera_front_base" args="1.75, 0.0, 1.32 -0.0 0.140 -0.0 /base_link /camera_front_base 100" />
    <node pkg="tf" type="static_transform_publisher" name="camera_front_base_to_img" args="0.0, 0.0, 0.0 -1.5707963267949 0.0 -1.5707963267949 /camera_front_base /camera_front_img 100" />
    
	<!--node pkg="vision_lane_detection" type="addCamInfo" name="addCamInfo">
	<param name="image_transport" value="compre5ssed"/>
	</node-->
    
    <node pkg="vision_lane_detection" type="ipm_pcl" name="ipm_pcl">
        <param name="image_transport" value="compressed"/>
	<!--remap from="image" to= "/camera_front/image_raw"/-->
	</node>

</launch>
